spinalcordtoolbox/scripts/sct_crop_image.py

```diff
@@ -18,19 +19,30 @@ def get_parser():
     parser = SCTArgumentParser(
         description="Tools to crop an image. Either via command line or via a Graphical User Interface (GUI). See "
                     "example usage at the end.",
-        epilog="EXAMPLES:\n"
-               "- To crop an image using the GUI (this does not allow to crop along the right-left dimension):\n"
-               "sct_crop_image -i t2.nii.gz -g 1\n\n"
-               "- To crop an image using a binary mask:\n"
-               "sct_crop_image -i t2.nii.gz -m mask.nii.gz\n\n"
-               "- To crop an image using a reference image:\n"
-               "sct_crop_image -i t2.nii.gz -ref mt1.nii.gz\n\n"
-               "- To crop an image by specifying min/max (you don't need to specify all dimensions). In the example "
-               "below, cropping will occur between x=5 and x=60, and between z=5 and z=zmax-1\n"
-               "sct_crop_image -i t2.nii.gz -xmin 5 -xmax 60 -zmin 5 -zmax -2\n\n"
-               "- To crop an image using a binary mask, and keep a margin of 5 voxels on each side in the x and y "
-               "directions only:\n"
-               "sct_crop_image -i t2.nii.gz -m mask.nii.gz -dilate 5x5x0\n\n"
+        epilog=textwrap.dedent("""
+            EXAMPLES:
+
+            - To crop an image using the GUI (this does not allow to crop along the right-left dimension):
+              ```
+              sct_crop_image -i t2.nii.gz -g 1
+              ```
+            - To crop an image using a binary mask:
+              ```
+              sct_crop_image -i t2.nii.gz -m mask.nii.gz
+              ```
+            - To crop an image using a reference image:
+              ```
+              sct_crop_image -i t2.nii.gz -ref mt1.nii.gz
+              ```
+            - To crop an image by specifying min/max (you don't need to specify all dimensions). In the example below, cropping will occur between x=5 and x=60, and between z=5 and z=zmax-1
+              ```
+              sct_crop_image -i t2.nii.gz -xmin 5 -xmax 60 -zmin 5 -zmax -2
+              ```
+            - To crop an image using a binary mask, and keep a margin of 5 voxels on each side in the x and y directions only:
+              ```
+              sct_crop_image -i t2.nii.gz -m mask.nii.gz -dilate 5x5x0
+              ```
+        """),  # noqa: E501 (line too long)
     )
 
     mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
```

Comment:
I've found a slight bug. The fact that we start every argument group with `\n` causes the anchor link for these sections to be rendered with an extra space (`' '`), which breaks `sphinx-argparse`.

Sidebar link (working):

![image](https://github.com/user-attachments/assets/db06749d-c984-4a7d-a370-c594d6f4b3b3)

Anchor link from `¶` hover button (not working):

![image](https://github.com/user-attachments/assets/d5887328-3c3d-4c51-9b28-cde0052ce917)

----

The good news, though, is... the `\n` actually have no effect on the terminal output! I'm not sure if Sphinx strips whitespace or what, but even after removing the `\n`, the output of `sct_script -h` renders identically (i.e. with a blank line).

So, I think I can go ahead and mass replace all instances of `parser.add_argument_group("\n` with `parser.add_argument_group("`.